src/main/java/org/broadinstitute/hellbender/tools/walkers/gnarlyGenotyper/GnarlyGenotyper.java

```diff
@@ -169,7 +172,9 @@ public boolean requiresReference() {
     @Override
     protected GenomicsDBOptions getGenomicsDBOptions() {
         if (genomicsDBOptions == null) {
-            genomicsDBOptions = new GenomicsDBOptions(referenceArguments.getReferencePath(), CALL_GENOTYPES, PIPELINE_MAX_ALT_COUNT);
+            genomicsdbArgs.callGenotypes = CALL_GENOTYPES;
+            genomicsdbArgs.maxDiploidAltAllelesThatCanBeGenotyped = PIPELINE_MAX_ALT_COUNT;
```

Comment:
Its a bit strange to mutate values in an argument collection like this (I know they aren't annotated as `@Argument`, but presumably someday they will be exposed ?). Can you verify that the only net change here in this part of the PR is that we're exposing the vcfCodec switch to `GnarlyGenotyper` (?).

Reply:
Yes, the net change here allows GnarlyGenotyper to access the vcfCodec arg. But, have also hooked `--max-genotype-count` arg from GenotypeGVCFs/GnarlyGenotyper to be passed to GenomicsDBOptions to help with Issue #6275.